tools/c7n_gcp/c7n_gcp/query.py

```diff
@@ -198,7 +198,14 @@ def _fetch_resources(self, query):
 
         for parent_instance in parent_resource_manager.resources():
             query.update(self._get_child_enum_args(parent_instance))
-            children = super(ChildResourceManager, self)._fetch_resources(query)
+
+            try:
+                children = super(ChildResourceManager, self)._fetch_resources(query)
+            except HttpError as e:
+                if e.resp.status != 400:
```

Comment:
i'm a little concerned that this possibly catches other things that might generate a 400, which is fairly generic as BadRequest error. this  handling seems fairly specific to sql database, so perhaps its best local to that resource manager.

Reply:
Thanks, good catch. Initially thought there would be at least one similar resource - spanner databases. But it turned out we cannot stop a Spanner instance.